api/app.ts

```diff
@@ -1,18 +1,20 @@
 import dotenv from 'dotenv';
 
 import express from 'express';
-import * as http from 'http';
-import * as winston from 'winston';
-import * as expressWinston from 'express-winston';
+import { Server, createServer } from 'http';
+import { transports, format } from 'winston';
+import { LoggerOptions, logger } from 'express-winston';
 import cors from 'cors';
 import debug from 'debug';
 import helmet from 'helmet';
-import CommonRoutesConfig from './common/common.routes.config';
-import UsersRoutes from './users/users.routes.config';
-
 import swaggerUi from 'swagger-ui-express';
+import { CommonRoutesConfig } from './common/common.routes.config';
+import { UsersRoutes } from './users/users.routes.config';
+
+import { HostsRoutes } from "./hosts/hosts.routes.config";
 // import swaggerDocument from './swagger.yaml';
```

Comment:
Commented code is forbidden (I don't remember, maybe it's mine? But it doesn't matter)